fabtests/test_configs/verbs/exclude

```diff
@@ -3,20 +3,16 @@
 # Exclude all prefix tests
 -k
 
+rdm
 dgram
```

Comment:
Can we remove `dgram` from exclude file? Or you see failed tests with verbs/dgram?

Reply:
It seems verbs dgram needs FI_MSG_PREFIX mode. I think the tests need to be updated to support it.